packages/x-components/.eslintrc.js

```diff
@@ -12,20 +12,7 @@ module.exports = {
   },
   overrides: [
     {
-      files: 'src/**/*.vue',
-      rules: {
-        'max-len': [
-          'error',
-          {
-            code: 100,
-            ignoreComments: false,
-            ignorePattern: '|class=".*"$ |https://github'
-          }
-        ]
-      }
-    },
-    {
-      files: 'src/components/icons/*.vue',
+      files: ['*.vue'],
       rules: {
         'max-len': 'off'
       }
```

Comment:
This is the only way I found to effectively disable `max-len`, I tried in eslint-x plugin file but it didn't have effect